src/test/java/core/basesyntax/operations/OperationsTest.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class OperationsTest {
+    private static final String CORRECT_TYPE = "P";
```

Comment:
it's better to test all correct types to be sure that each of them works.

Reply:
done